bundles/org.openhab.binding.snmp/README.md

```diff
@@ -0,0 +1,141 @@
+# SNMP Binding
+
+This binding integrates the Simple Network Management Protocol (SNMP).
+SNMP can be used to monitor or control a large variety of network equipment, e.g. routers, switches, NAS-systems.
+Currently protocol version 1 and 2c are supported.
+
+## Supported Things
+
+Only one thing is supported: `target`.
+It represents a single network device. 
+Things can be extended with `number`, `string` and `switch` channels.
+
+## Binding Configuration
+
+If only GET/SET-requests shall be used, no configuration is necessary.
+In this case the `port` parameter defaults to `0`.
+
+For receiving traps a port for receiving traps needs to be configured.
+The standard port for receiving traps is 162, however binding to ports lower than 1024 is only allowed with privileged right on most *nix systems.
+Therefore it is recommended to bind to a port higher than 1024 (e.g. 8162).
+In case the trap sending equipment does not allow to change the destination port (e.g. Mikrotik routers), it is necessary to forward the received packets to the new port.
+This can be done either by software like _snmptrapd_ or by adding a firewall rule to your system, e.g. by executing
+
+```
+iptables -t nat -I PREROUTING --src 0/0 --dst 192.168.0.10 -p udp --dport 162 -j REDIRECT --to-ports 8162
+```
+
+would forward all TCP packets addressed to 192.168.0.10 from port 162 to 8162. 
+Check with your operating system manual how to make that change permanent. 
+
+Example configuration for using port 8162:
+
+```
+# Configuration for the SNMP Binding
+#
+# Port used for receiving traps.
+# This setting defaults to 0 (disabled / not receiving traps)
+port=8162
+
+```
+
+## Thing Configuration
+
+The `target` thing has one mandatory parameter: `hostname`.
+It can be set as FQDN or IP address. 
+
+Optional configuration parameters are `community`, `version` and `refresh`.
+
+The SNMP community can be set with the `community` parameter.
+It defaults to `public`.
+
+Currently two protocol versions are supported.
+The protocol version can be set with the `protocol` parameter.
+The allowed values are `0` for v1 and `1` for v2c.
```

Comment:
Why is this so indirect? Can't we just use strings to configure, using 1 to confgure v2 sounds a bit to nerdy to me :-)